### ISSUE TITLE
Suddenly getting error about "workbox-build" version

### ISSUE THREAD

**Issue by iamdevlinph** [vercel/vercel#3646]
Previously working build log

```
Downloading 220 deployment files...
Installing build runtime...
Build runtime installed: 514.247ms
Looking up build cache...
Build cache found [34.40 MB, 40341 files]
Build cache unpacked: 2224.450ms
Installing dependencies...
yarn install v1.17.3
warning package.json: No license field
warning package.json: "dependencies" has dependency "@types/flat" with range "^0.0.28" that collides with a dependency in "devDependencies" of the same name with version "0.0.28"
info No lockfile found.
warning payroll-client@1.0.0: No license field
warning payroll-client@1.0.0: "dependencies" has dependency "@types/flat" with range "^0.0.28" that collides with a dependency in "devDependencies" of the same name with version "0.0.28"
[1/4] Resolving packages...
warning next > @babel/runtime-corejs2 > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning next > styled-jsx > babel-types > babel-runtime > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning next-offline > workbox-webpack-plugin > workbox-build > rollup-plugin-node-resolve@5.2.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-node-resolve.
warning next-offline > workbox-webpack-plugin > workbox-build > rollup-plugin-replace@2.2.0: This module has moved and is now available at @rollup/plugin-replace. Please update your dependencies. This version is no longer maintained.
warning @types/axios@0.14.0: This is a stub types definition for axios (https://github.com/mzabriskie/axios). axios provides its own type definitions, so you don't need @types/axios installed!
warning @types/localforage@0.0.34: This is a stub types definition for localforage (https://github.com/localForage/localForage). localforage provides its own type definitions, so you don't need @types/localforage installed!
warning @types/next@9.0.0: This is a stub types definition. next provides its own type definitions, so you do not need this installed.
warning @types/react-datepicker > popper.js@1.16.1: Popper changed home, find its new releases at @popperjs/core
[2/4] Fetching packages...
info fsevents@2.1.2: The platform "linux" is incompatible with this module.
info "fsevents@2.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@1.2.11: The platform "linux" is incompatible with this module.
info "fsevents@1.2.11" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning "@zeit/next-css > css-loader@1.0.0" has unmet peer dependency "webpack@^4.0.0".
warning "@zeit/next-css > mini-css-extract-plugin@0.4.3" has unmet peer dependency "webpack@^4.4.0".
warning "styled-components > stylis-rule-sheet@0.0.10" has unmet peer dependency "stylis@^3.5.0".
warning " > next-offline@5.0.0-beta.10" has unmet peer dependency "webpack@^4.19.1".
warning "next-offline > workbox-webpack-plugin@5.0.0-rc.2" has unmet peer dependency "webpack@^4.0.0".
warning "next-offline > copy-webpack-plugin@5.0.5" has unmet peer dependency "webpack@^4.0.0".
warning " > styled-components@5.0.0-rc.2" has unmet peer dependency "react-is@>= 16.8.0".
warning "@zeit/next-typescript > @babel/preset-typescript@7.8.3" has unmet peer dependency "@babel/core@^7.0.0-0".
[4/4] Building fresh packages...
success Saved lockfile.
Done in 22.05s.
Running "yarn run now-build"
```

Current log with error

```
Downloading 220 deployment files...
Installing build runtime...
Build runtime installed: 638.993ms
Looking up build cache...
Build cache found [34.75 MB, 40772 files]
Build cache unpacked: 2186.513ms
Installing dependencies...
yarn install v1.17.3
warning package.json: No license field
info No lockfile found.
warning payroll-client@1.0.0: No license field
[1/4] Resolving packages...
warning next > @babel/runtime-corejs2 > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning next > styled-jsx > babel-types > babel-runtime > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
error Couldn't find any versions for "workbox-build" that matches "^5.0.0-rc.3"
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: Exited with 1
    at ChildProcess.child.on (/zeit/93391daef1d6236b/.build-utils/.builder/node_modules/@now/next/dist/index.js:47107:24)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
worker exited with code 20 and signal null
done
```

**Comment by styfle** [comment]
Hi @iamdevlinph 

The error message is correct.

> error Couldn't find any versions for "workbox-build" that matches "^5.0.0-rc.3"

There is no version `5.0.0-rc.3` of `workbox-build` as seen on [npmjs.com](
https://www.npmjs.com/package/workbox-build?activeTab=versions).

![image](https://user-images.githubusercontent.com/229881/73028971-a65b1600-3e04-11ea-921c-7775e6f74a5b.png)

**Comment by iamdevlinph** [comment]
The problem is that we don't have `workbox-build` as a dependency nor in the yarn.lock. I also tried deploying a very old working commit but still no.

**Comment by iamdevlinph** [comment]
The closes thing we have to a `workbox` is in the `next.config.js` but regardless of this, deployment was working before.
```js
const withOffline = require('next-offline');
const withCSS = require('@zeit/next-css');
const config = require('./config');

const nextConfig = {
  target: 'serverless',
  transformManifest: manifest => ['/'].concat(manifest),
  generateInDevMode: false,
  workboxOpts: {
    swDest: 'static/service-worker.js',
    runtimeCaching: [
      {
        urlPattern: /^api?.*/,
        handler: 'NetworkFirst',
        options: {
          cacheName: 'https-calls',
          networkTimeoutSeconds: 15,
          expiration: {
            maxEntries: 150,
            maxAgeSeconds: 30 * 24 * 60 * 60, // 1 month
          },
          cacheableResponse: {
            statuses: [0, 200],
          },
        },
      },
    ],
  },
  env: {
    apiUrl: config.apiUrl,
  },
};

module.exports = withCSS(withOffline(nextConfig));

```

**Comment by iamdevlinph** [comment]
Found the cause of the issue.

Replaced ~`"next-offline": "5.0.0-beta.10",`~ with `"next-offline": "4.0.6",`

The only weird thing is that this has been working before.

**Comment by iamdevlinph** [comment]
Will also be fixed by updating to `"next-offline": "5.0.0-beta.11",`
